app/api/payments/mpesa/route.ts

```
@@ -55,6 +71,20 @@ export async function POST(request: NextRequest) {
     logger.debug(
       `Authorization Header (masked): ${authHeader.substring(0, 20)}...`,
     ); // Mask for security
+    logger.debug(
+      `MPESA_BUSINESS_SHORT_CODE: ${process.env.MPESA_BUSINESS_SHORT_CODE}`,
+    );
+    logger.debug(
+      `MPESA_PASSKEY (masked): ${process.env.MPESA_PASSKEY ? process.env.MPESA_PASSKEY.substring(0, 5) + "..." : "not set"}`,
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)